### PR TITLE
Recommended npm upgrade script on Windows

### PR DESCRIPTION
**Pull request by danilosampaio**
Added `npm-windows-upgrade` tool as the recommended npm upgrade script on Windows.

### CODE REVIEW

**Comment by SBoudrias** [comment]
Linking to the guide seems like a first good step. Tools are good, but it's quite an overhead.

So, I'd rephrase this line to be less "we recommend" and more "an option is" to use that tool. And I'd move it after the documentation links.

**Comment by danilosampaio** [comment]
Hi @SBoudrias 

like that?

See this guide if you're having trouble upgrading:
https://github.com/npm/npm/wiki/Troubleshooting#upgrading-on-windows
Upgrading npm on Windows requires some manual steps, that can be easier by using the npm-windows-upgrade tool:
https://github.com/felixrieseberg/npm-windows-upgrade

**Comment by SBoudrias** [comment]
Maybe "You might want to use npm-windows-upgrade to simplify the updating process"

**Comment by SBoudrias** [comment]
Actually, I just realized the npm documentation link to this tool directly at the top.

I think this is plenty enough for windows user.
